docs/streams/developer-guide/config-streams.html

```diff
@@ -441,8 +441,8 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
           </tr>
           <tr class="row-even"><td>task.assignor.class</td>
             <td>Medium</td>
-            <td colspan="2">A task assignor class or class name implementing the <code>org.apache.kafka.streams.processor.assignment.TaskAssignor</code> interface.</td>
-            <td>The default high-availability task assignor.</td>
+            <td colspan="2">A task assignor class or class name implementing the <code>TaskAssignor</code> interface.</td>
```

Comment:
should we maybe leave this as is (with the full package name)? We decided to leave the old assignors and old TaskAssignor interface around for 3.8 as a sort of "feature flag"/ backdoor in case there's an issue with the new assignor. So right now there are technically two TaskAssignor interfaces

It might be fine since only the new one is in a public package, but I know people don't always pay the best attention to  whether something is in an internal package or not

Reply:
alternatively we might just want to rename the old interface to `LegacyTaskAssignor` or something like that. See my message on CC slack cc @mjsax